src/Orangebeard.Client/Entities/Attachment.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.IO;
+
+namespace Orangebeard.Client.Entities
+{
+    public class Attachment
```

Comment:
should we rename this to LogWithAttachment?

Reply:
In the Java client, it is also simply called `Attachment`. 
Although the class has a field `Message`, this is usually just used to pass the attachment's file name, not an actual log message.